src/api/stats/heco/getHfiApys.js

```diff
@@ -0,0 +1,52 @@
+const axios = require('axios');
+const https = require('https');
+
+const pools = require('../../../data/heco/hfiPools.json');
+const { compound } = require('../../../utils/compound');
+const { BASE_HPY } = require('../../../constants'); 
+
+
+const getHfiApys = async () => {
+  let apys = {};
+
+  const hfiStats = await fetchHfiStats();
+  
+
+  const values = pools.map(pool => {
+        const poolStat = hfiStats[pool.poolId];
+
+    const vaultApy = Number(poolStat['annual']);
+    const vaultDec = (vaultApy / 100);
+    const aprHfi = Number(poolStat['hfi_annual']);
+    const aprDec = (aprHfi / 100);
+    const hfiApy = compound(aprDec * 0.955, BASE_HPY, 1);

```

Comment:
I'm guessing this .955 is accounting for the beefy performance fee? Can you use a constant variable for this?